shared/naturalcrit/codeEditor/fold-css.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+	registerHomebreweryHelper : function(CodeMirror) {
+		CodeMirror.registerHelper('fold', 'homebrewerycss', function(cm, start) {
+
+			// BRACE FOLDING
+			const startMatcher = /\{[ \t]*$/;
+			const endMatcher = /\}[ \t]*$/;
+			const prevLine = cm.getLine(start.line);
```

Comment:
The `prevLine` naming is somewhat confusing; I believe that this came from the `fold-code.js`, but the folding blocks in that module are quite different to these CSS blocks. This could be renamed to something clearer: `startLine` or something similar.

Reply:
It did indeed.